marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/entities/search/EntitySearchManagerTest.java

```diff
@@ -70,37 +70,29 @@ void searchWithTransformInStagingDatabase() {
     }
 
     @Test
-    void noEntityTypesSelected() {
-        runAsDataHubDeveloper();
-        // Need at least one entity model to exist for this scenario
-        installOnlyReferenceModelEntities(true);
-
-        runAsHubCentralUser();
-
-        SearchQuery query = new SearchQuery();
-        DocSearchQueryInfo info = new DocSearchQueryInfo();
-        info.setEntityTypeIds(Arrays.asList(" "));
-        query.setQuery(info);
+    void searchAllDataInStagingDatabase() {
+        validateAllDataSearch("staging");
+    }
 
-        String results = new EntitySearchManager(getHubClient(), "staging").search(query).get();
-        ObjectNode node = readJsonObject(results);
-        assertEquals(0, node.get("total").asInt(), "When entityTypeIds has values, but they're all empty strings, the " +
-            "backend should return no results, and not throw an error");
+    @Test
+    void searchAllDataInFinalDatabase() {
+        validateAllDataSearch("final");
     }
 
     @Test
     public void testSearchResultsOnNoData() {
         runAsDataHubDeveloper();
-        EntitySearchManager.QUERY_OPTIONS = "non-existent-options";
         String collectionDeleteQuery = "declareUpdate(); xdmp.collectionDelete(\"http://marklogic.com/entity-services/models\")";
         getHubClient().getFinalClient().newServerEval().javascript(collectionDeleteQuery).evalAs(String.class);
 
         runAsHubCentralUser();
-        assertTrue(new EntitySearchManager(getHubClient()).search(new SearchQuery()).get().isEmpty());
+        StringHandle results = new EntitySearchManager(getHubClient()).search(new SearchQuery());
+        ObjectNode node = readJsonObject(results.get());
+        assertEquals(14, node.get("total").asInt());
 
         SearchQuery query = new SearchQuery();
         query.getQuery().setEntityTypeIds(Arrays.asList("Some-entityType"));
-        assertTrue(new EntitySearchManager(getHubClient()).search(query).get().isEmpty());
+        assertThrows(DataHubException.class, () -> new EntitySearchManager(getHubClient()).search(query));
```

Comment:
This needs an assertion message too - why do we expect this exception? And what's the point of a DataHubException - what value is that adding over the original exception? 

I noticed that class is used in 7 places, but there's no code that cares about DataHubException. For that reason, I recommend removing it. It's needlessly wrapping the original exception but not providing any value.